### PR TITLE
Fix validation error and apiURL

### PR DESCRIPTION
**Pull request by bbergshaven**

### CODE REVIEW

File: nodes/clarify_insert.js [hunk 1]
```diff
@@ -148,6 +148,12 @@ module.exports = function (RED) {
     this.on('input', async function (msg, send, done) {
       // Validate incoming id. Must be correct to continue
       let id = RED.util.getMessageProperty(msg, config.signalId);
+      if (id === undefined) {
+        let errMsg = 'missing signal id';
+        node.status({fill: 'red', shape: 'ring', text: errMsg});
+        done(errMsg);
+        return;
+      }
       if (!signalIdPattern.test(id)) {
```
Comment (khtufteland) [inline nodes/clarify_insert.js:157]: Didn't this pickup `id === undefined`?

**Review by khtufteland — APPROVED** [review]
You have two commits in this PR, one empty and one with the errors.
I always add the git log in the description using the command:
`git log -"$(git rev-list --count HEAD ^master)" | pbcopy`.

File: nodes/clarify_insert.js (hunk 1 above)
Comment (bbergshaven) [reply nodes/clarify_insert.js:157]: nope, it did not. :/

Comment (khtufteland) [reply nodes/clarify_insert.js:157]: Weird. Anyways. Nice to get an explicit error message when the id isn't defined.

**Pull request merged**